test/functional/mail_notification_test.rb

```diff
@@ -0,0 +1,42 @@
+require File.join(File.dirname(__FILE__), 'test_helper')
+
+describe 'mail_notification' do
+  describe 'list' do
+    before do
+      @cmd = %w(mail-notification list)
+      @mail_notifications = [{
+        id: 1,
+        name: 'config_summary',
+        description: 'A summary of eventful configuration management reports',
+        subscription_type: 'report'
+      }]
+    end
+
+    it 'should return a list of mail notifications' do
+      api_expects(:mail_notifications, :index, 'List mail notifications').returns(@mail_notifications)
+
+      result = run_cmd(@cmd)
+      result.exit_code.must_equal HammerCLI::EX_OK
```

Comment:
The same instead of checking for exit code, but for list commands: https://github.com/theforeman/hammer-cli-foreman/blob/master/test/functional/personal_access_token_test.rb#L43-L51